organization/mission.md

```diff
@@ -22,9 +22,15 @@ The founding principles were adopted by the board of directors on 23 August 2019
 
 \* In order to start as an association 4 initial individual founders will play the role of member until there are enough public members for the organization to sustain itself.
 
-## Goals
+## Current priorities
 
-* Enable an **ecosystem of Public Code software products**, developed by and for public administrations to solve real problems for citizens
-* Build a **collaborative community of developers and cities**, which continuously improves and maintains Public Code products
-* Provide a **certification system for software** as meeting the standard set by the Foundation For Public Code
-* Build a **trusted brand and an awareness of the value of Public Code**
+The current priorities of the organization are set
+by the [board of directors](governance-model.md)
+and reviewed every meeting.
+
+29 April 2020
```

Comment:
By putting this on a separate line it can be changed independently without adding a lot of noise to the rest of the commit. An update of the priorities would then have this change and the updated priorities.